wp-sw-cache/wp-sw-cache-main.php

```diff
@@ -9,11 +10,7 @@ class SW_Cache_Main {
   private static $instance;
 
   public function __construct() {
-    if($this->show_header()) {
```

Comment:
As you can see, I removed this check cause adding the registrar is automatically performed by the library but this also means the worker is being registered on each access. I need to double check this behavior to ensure it is correct.